flask_project/campaign_manager/models/models.py

```diff
@@ -334,6 +350,18 @@ def get_task_boundary_as_geoJSON(self):
             TaskBoundary.campaign_id == self.id
             ).first()
 
+    def get_participant_count(uuid):
+        """ Returns the participant count for the campaign """
+        participant_count = 0
+        campaign = session.query(Campaign).filter(
```

Comment:
you could copy/paste the session.query code into a method called `find_by_uuid` on the campaign object.